ddtrace/contrib/aiohttp/patch.py

```diff
@@ -26,7 +28,7 @@ def patch():
 
         _w = wrapt.wrap_function_wrapper
         _w('aiohttp_jinja2', 'render_template', _trace_render_template)
-        Pin(app='aiohttp', service=None, app_type='web').onto(aiohttp_jinja2)
+        Pin(app='aiohttp', service=None, tracer=tracer, app_type='web').onto(aiohttp_jinja2)
```

Comment:
Same issue explained in #479 . We should keep another approach. Also these changes should be merged in `opentracer` branch and not on `master`, otherwise they will be shipped in the next bugfix release.